### PR TITLE
teams/migration: a vote for joccau as the reviewer

### PR DESCRIPTION
**Pull request by 3pointer**
As title.

### CODE REVIEW

**Review by YuJuncen — APPROVED** [review]
LGTM

**Review by kennytm — APPROVED** [review]

**Comment by kennytm** [comment]
since the `pingcap/tidb` and `pingcap/br` repos are governed by team/tidb should this change be applied to team/tidb too? (ptal @zhangyangyu?)

File: teams/migration/membership.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
         "gozssky",
         "iamxy",
         "okJiang",
+        "joccau",
```
Comment (kennytm) [inline teams/migration/membership.json:44]: oh please keep the new names in alphabetical order.

**Comment by zhangyangyu** [comment]
> since the `pingcap/tidb` and `pingcap/br` repos are governed by team/tidb should this change be applied to team/tidb too? (ptal @zhangyangyu?)

We have a dedicated tidb/migration team, so I think add him to tidb/migration team is enough. If the migration team also needs to contribute to tidb repo, I'd suggest to add tidb repo to migration team.

**Pull request merged**